lib/TraackrApi/Influencers.php

```diff
@@ -2,357 +2,353 @@
-         implode(',', $p['tags']) : $p['tags'];
-
-      return $inf->post(TraackrApi::$apiBaseUrl.'influencers/tag/add', $p);
-
-   }
-
-   /**
-    * Remove a tag from an influencer
-    *
-    * @param array $p
-    * @return bool|mixed
-    * @throws MissingParameterException
-    */
-   public static function tagRemove($p = array('all' => false)) {
-
-      $inf = new Influencers();
-
-      // Sanitize default values
-      $p['all'] = $inf->convertBool($p, 'all');
-
-      $p = $inf->addCustomerKey($p);
-      // 'influencers' is not required if 'all' is set to true
-      // by then 'all' has already be converted to a string
-      if ( $p['all'] === 'false' ) {
-         $inf->checkRequiredParams($p, array('influencers', 'tags', 'customer_key', 'all'));
-      } else {
-         $inf->checkRequiredParams($p, array('tags', 'customer_key', 'all'));
-      }
-
-      // support for multi params
-      if ( !empty($p['influencers']) ) {
-         $p['influencers'] = is_array($p['influencers']) ?
+class Influencers extends TraackrApiObject
+{
+    /**
+     * Get an influencer's data
+     *
+     * @param string $uid
+     * @param array $p
+     * @return bool|mixed
+     * @throws MissingParameterException
+     */
+    public static function show($uid, $p = array('with_channels' => false))
+    {
+        if (empty($uid)) {
+            throw new MissingParameterException("Missing Influencer UID parameter");
+        }
+
+        // API Object
+        $inf = new Influencers();
+
+        //Sanitize default values
+        $p['with_channels'] = $inf->convertBool($p, 'with_channels');
+
+        // Add customer key + check required params
+        $p = $inf->addCustomerKey($p);
+        $inf->checkRequiredParams($p, array('with_channels'));
+        // support for multi params
+        $uid = is_array($uid) ? implode(',', $uid) : $uid;
+
+        return $inf->get(TraackrApi::$apiBaseUrl . 'influencers/show/' . $uid, $p);
+    }
+
+    /**
+     * Returns an influencer's connections
+     *
+     * @param string $uid
+     * @param string $direction
+     * @return bool|mixed
+     * @throws MissingParameterException
+     */
+    public static function connections($uid, $direction = '')
+    {
+        if (empty($uid)) {
+            throw new MissingParameterException("Missing Influencer UID parameter");
+        }
+
+        $uid = is_array($uid) ? implode(',', $uid) : $uid;
+        $direction = empty($direction) ? '' : $direction . '/';
+        $inf = new Influencers();
+        return $inf->get(TraackrApi::$apiBaseUrl . 'influencers/connections/' . $direction . $uid, []);
+    }
+
+    /**
+     * Lookup Influencer by a Twitter handle
+     *
+     * @param string $username
+     * @param string $type USERNAME | TWITTER_ID
+     * @return bool|mixed
+     * @throws MissingParameterException
+     * @throws \UnexpectedValueException
+     */
+    public static function lookupTwitter($username, $type = 'USERNAME')
```

Comment:
New parameter `$type`

Reply:
I guess it's alright to default to USERNAME rather than just blank, which is the same thing (I double checked; API accepts blank).